### PR TITLE
Redirect http://bazel.io to https://bazel.io.

### LINKED ISSUE #1644: http://bazel.io should redirect to https://bazel.io
https://bazel.io is now live, but http://bazel.io does not redirect to the https site.

### PR DESCRIPTION
**Pull request by davidzchen**
Fixes #1644

### CODE REVIEW

**Comment by davidzchen** [comment]
+cc @damienmg

File: site/_includes/head.html [hunk 1]
```diff
@@ -8,7 +8,11 @@
       var current_url = window.location.href;
       var bad_url = new RegExp("^https?://bazelbuild.github.io/bazel/");
       if (bad_url.test(current_url)) {
-        window.location.replace(current_url.replace(bad_url, "http://bazel.io/"));
+        window.location.replace(current_url.replace(bad_url, "https://bazel.io/"));
```
Comment (damienmg) [inline site/_includes/head.html]: replace to www.bazel.io?

**Comment by damienmg** [comment]
LGTM. I would just recommend to use https://www.bazel.io as the canonical url

**Comment by davidzchen** [comment]
Done.

**Comment by damienmg** [comment]
LGTM

**Pull request closed without merging**